src/main/java/com/moviecat/model/Genre.java

```diff
@@ -0,0 +1,96 @@
+/*
+ *  Licensed to the MovieCat, Inc. under one
+ *  or more contributor license agreements. See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.
+ *
+ *  You may not use this file except in compliance
+ *  with the License.
+ *
+ */
+
+package com.moviecat.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import dev.morphia.annotations.Id;
+
+import javax.annotation.Generated;
+import java.util.Objects;
+
+public class Genre {
```

Comment:
Annotate the class with `@Entity` so that Mongo knows to create a collection for it.

Reply:
Great